docs/source/developer/css.rst

```diff
@@ -191,3 +194,40 @@ only the desired children.
 
 When in doubt, there is little harm done in parents adding selectors to
 children.
+
+Commonly useful CSS selectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following common CSS class names are treated as public.
```

Comment:
"CSS selectors"

Reply:
I would go a bit further in describing what is meant by public here. In particular, we use CSS selectors when deciding what command to invoke for keyboard shortcuts, and for what context menu items to show upon right-click. So if extension authors want to add keyboard shortcuts and context menu items, they will want to use our CSS classes. Here we are specifically flagging ones that we intend to be used.